test/test/util/jasmine_fetch.js

```diff
@@ -134,7 +134,7 @@ jasmine.Fetch.impl_ = function(url, init) {
     });
     return Promise.resolve(response);
   } else if (stubbed.error) {
-    return Promise.reject('fake error');
+    return Promise.reject({ message: 'fake error' });
```

Comment:
I made this change to make it easy to expect `jasmine.any(Object)` when checking the error.